src/main/java/baza/trainee/controller/ArticleController.java

```diff
@@ -0,0 +1,37 @@
+package baza.trainee.controller;
+
+import baza.trainee.domain.dto.ArticleDto;
+import baza.trainee.service.ArticleService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+/**
+ * Spring MVC REST controller serving article operations for non-admin users.
+ *
+ * @author Olha Ryzhkova
+ * @version 1.0
+ */
+@RestController
+@RequestMapping("/api/article")
+@RequiredArgsConstructor
+public class ArticleController {
```

Comment:
We also need a non-secure endpoint for retrieving the page of articles.

Reply:
For now, we don't have a business case for using the page of articles. I suggest adding this endpoint later if necessary.